PlaygroundService/Grains/PlaygroundGrain.cs

```diff
@@ -19,6 +19,44 @@ public class PlaygroundGrain : Grain, IPlaygroundGrain
         _logger = logger;
     }
 
+    public async Task <bool> DelData(string zipFile, string dllPath)
```

Comment:
dllPath is actually the extractedPath? It is not just the path to the dll but the whole extracted folder from the zip. If this is the case, please rename dllPath.